packages/cashier/src/stores/withdraw-store.ts

```diff
@@ -291,31 +295,27 @@ export default class WithdrawStore {
             });
             if (!is_ok) error_message = message;
             else if (+balance < +converter_from_amount) error_message = localize('Insufficient funds');
-            else if (+balance < +min_withdraw_amount) {
+            else if (min_withdraw_amount !== undefined && +balance < +min_withdraw_amount) {
```

Comment:
can we use  `else if (min_withdraw_amount && +balance < +min_withdraw_amount) `?

Reply:
@vinu-deriv I would also suggest replacing `+n` with `Number(n)` for better readability.